docs/examples/datamover_advanced_voloptions.md

```diff
@@ -0,0 +1,374 @@
+```
+
+- Wait several minutes and check the VolumeSnapshotBackup CR status for `completed`: 
+
+- There should now be a snapshot(s) in the object store that was given in the restic secret.
+- You can check for this snapshot in your targeted `backupStorageLocation` with a
+prefix of `/<OADP-namespace>`
+
+<h4> For Restore <a id="restore"></a></h4>
+
+- Make sure the application namespace is deleted, as well as any volumeSnapshotContents
+  that were created during backup.
+
+- Create a restore CR:
+
+```yml
+apiVersion: velero.io/v1
+kind: Restore
+metadata:
+  name: <restore-name>
+  namespace: <protected-ns>
+spec:
+  backupName: <previous-backup-name>
+```
+- Monitor the datamover backup and artifacts via [a debug script](/docs/examples/debug.md)
+OR
+- Check the `VolumeSnapshotRestore`(s) progress: 
+
+```
+oc get vsr -n <app-ns>
+oc get vsr <vsr-name> -n <app-ns> -ojsonpath="{.status.phase}
+```
+
+- Check that your application data has been restored:
+
+`oc get route <route-name> -n <app-ns> -ojsonpath="{.spec.host}"`
+
+
+<h1 align="center">Backup/Restore with Split Volumes: CephFS and CephRBD<a id="fsrbd"></a></h1>
+
+- Ensure a stateful application is running in a separate namespace with PVCs provisioned
+  by both CephFS and CephRBD
+
+- This assumes cephFS is being used as the default `StorageClass` and 
+    `VolumeSnapshotClass`
+
+- Create a DPA similar to below:
+  - Add the restic secret name from the prerequisites to your DPA CR in 
+  `spec.features.dataMover.credentialName`. If this step is not completed then 
+    it will default to the secret name `dm-credential`
+  - Note: `volumeOptionsForStorageClass` can be defined for multiple storageClasses,
+    thus allowing a backup to complete with volumes with different providers.
+
+```yml
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  name: velero-sample
+  namespace: openshift-adp
+spec:
+  backupLocations:
+    - velero:
+        config:
+          profile: default
+          region: us-east-1
+        credential:
+          key: cloud
+          name: cloud-credentials
+        default: true
+        objectStorage:
+          bucket: <my-bucket>
+          prefix: velero
+        provider: aws
+  configuration:
+    restic:
+      enable: false
+    velero:
+      defaultPlugins:
+        - openshift
+        - aws
+        - csi
+        - vsm
+  features:
+    dataMover:
+      credentialName: <restic-secret-name>
+      enable: true
+      volumeOptionsForStorageClasses:
+        ocs-storagecluster-cephfs:
+          sourceVolumeOptions:
+            accessMode: ReadOnlyMany
+            cacheAccessMode: ReadWriteMany
+            cacheStorageClassName: ocs-storagecluster-cephfs
+            moverSecurityContext: true
+            storageClassName: ocs-storagecluster-cephfs-shallow
+          destinationVolumeOptions:
+            moverSecurityContext: true
+        ocs-storagecluster-ceph-rbd:
```

Comment:
@eemcmullan @weshayutin Why do we need ceph rbd in volumeoptions mapping ? Are we trying to override something for rbd ? if not then why specify ?

Reply:
It'll work either way. I added it to show that field was there, but can remove

Reply:
yeah I agree it will work but lets this whole feature is to override options if there is a diff in the needed options vs the default options. So lets remove it so that the users do not get confused IMO.